BGL/doc/BGL/PackageDescription.txt

```diff
@@ -708,7 +708,7 @@ user might encounter.
 ## I/O Functions ##
 - \link PkgBGLIOFct CGAL::read_off() \endlink
 - \link PkgBGLIOFct CGAL::write_off() \endlink
-- \link PkgBGLIOFct CGAL::write_VTP() \endlink
+- \link PkgBGLIOFct CGAL::write_vtp() \endlink
```

Comment:
@sloriot Why do we use here 
```
- \link PkgBGLIOFct .. \endlink
```
instead of just:
```
- `CGAL::write_vtp`
```
?

Reply:
It is needed read/write off because there are different overloads in different packages. For `write_vtp` it is not needed IMO.